### PR TITLE
Updates for a release in Kinetic

### PR DESCRIPTION
**Pull request by bmagyar**
For some reason tests don't show up for me, hoping it's a local issue and Travis gets them right.

The folder `tests` is mostly rosbuild. Are those ran anywhere?

### CODE REVIEW

**Comment by bmagyar** [comment]
Please don't merge this yet. I'd like to look around for cleaning-up possibilities. Feel free to post suggestions too.

**Comment by bmagyar** [comment]
I've spotted that the local `indigo-devel` diverged from `kinetic-devel` so cherry-picked those commits on top of my changes as it'd be a bit harsh to rebase the `kinetic-devel` branch now.

**Comment by bmagyar** [comment]
Guys, this really needs a review now :) 

@v4hn @130s @davetcoleman

**Comment by davetcoleman** [comment]
Not sure why this "really needs a review" but ok.

File: tests/pr2_jacobian_tests/src/main.cpp
Comment (davetcoleman) [inline tests/pr2_jacobian_tests/src/main.cpp:1]: Why not keep these tests and just make them work with Catkin?

**Comment by bmagyar** [comment]
I meant that by the timing. It's been ready for a while

**Comment by bmagyar** [comment]
We actually have that very test migrated to Kinetic in the moveit_pr2_tests
package. The rogue rosbuild ones in `tests` will require some attention
later as they are from the `arm_navigation` era.

**Comment by davetcoleman** [comment]
Where is moveit_pr2_tests? Seems like it should be in moveit_pr2 so that any changes get properly tested.

**Comment by bmagyar** [comment]
My bad. It is `pr2_moveit_tests` and lives in this repo.

File: tests/pr2_jacobian_tests/src/main.cpp
Comment (bmagyar) [reply tests/pr2_jacobian_tests/src/main.cpp:1]: The email response didn't get the message to the right place, see https://github.com/ros-planning/moveit_pr2/pull/96#issuecomment-382352007 for reply.

**Comment by bmagyar** [comment]
@130s could you please add your feedback? I'm only lacking that to push a release.

File: moveit_pr2/package.xml
Comment (130s) [inline moveit_pr2/package.xml]: May not be a big deal but we probably want to remove these 2 maintainers.

File: moveit_pr2/package.xml
Comment (130s) [inline moveit_pr2/package.xml]: Alphabetical order of the entries, if we are too concerned.

File: pr2_moveit_plugins/package.xml
Comment (130s) [inline pr2_moveit_plugins/package.xml]: May not be a big deal but we probably want to remove these 2 maintainers.

**Review by 130s — APPROVED** [review]
Added only minor comments that don't affect merge decision.

File: moveit_pr2/package.xml
Comment (bmagyar) [reply moveit_pr2/package.xml]: Resolved

File: moveit_pr2/package.xml
Comment (bmagyar) [reply moveit_pr2/package.xml]: Resolved

File: pr2_moveit_plugins/package.xml
Comment (bmagyar) [reply pr2_moveit_plugins/package.xml]: Resolved

**Comment by bmagyar** [comment]
Thanks for the feedback guys!

**Pull request merged**
